libs/checkpoint/langgraph/checkpoint/sqlite/utils.py

```diff
@@ -99,6 +68,9 @@ def search_where(
     if config is not None:
         wheres.append("thread_id = ?")
         param_values.append(config["configurable"]["thread_id"])
+        checkpoint_ns = config["configurable"].get("checkpoint_ns", "")
```

Comment:
I think when you finish up nested checkpoints we'll want the ability to call `list` and get back checkpoints for all namespace ids

Reply:
sounds good to me